apps/coscrad-frontend/src/components/commands/ack-notification.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button, Card, CardContent } from '@mui/material';
+
+interface AckNotificationProps {
+    _onClick: () => void;
+}
+
+export const AckNotification = ({ _onClick: onClick }: AckNotificationProps): JSX.Element => (
+    <Card>
```

Comment:
We could consider using the Basic Dialog from MUI for this message: https://mui.com/material-ui/react-dialog/

Reply:
[Here](https://www.pivotaltracker.com/story/show/184132185) is the story.